index.html

```diff
@@ -530,6 +608,33 @@ <h3>StatusList2021Credential</h3>
   "proof": { ... }
 }
         </pre>
+        <pre class="example nohighlight" title="Example StatusList2021Credential">
```

Comment:
<p class="issue" data-number="73" title="Design of multiple status messages is not finalized.">
The Working Group is still discussing the unification of a design between status lists with a single state (such as "revoked" or "suspended") and status lists with multiple states (exposed via a series of status messages). We are seeking implementer feedback on what a unified design should look like from an ease of implementation, privacy, and security standpoint.
</p>